### DISCUSSION TITLE
Add number of snippets in front of each category

### DISCUSSION THREAD

**Discussion by XNinety9** [massCodeIO/massCode#108] [Ideas]
Hi.

I'd like to file a feature request: could you add number of snippets contained in each category?

Here is a crappy simulation of what I mean:

![untitled](https://user-images.githubusercontent.com/4655239/177723742-d3431f7d-5e8a-4dee-b78b-69c88768b1ea.jpg)

Thanks a lot for massCode

**Comment by antonreshetov** [comment]
Hi @p-vernaeckt

I'm just wondering, what's the benefit to the user?

**Comment by XNinety9** [comment]
Just having an idea of how much content there is in each category.
